src/multiplots.ts

```diff
@@ -1293,15 +1278,28 @@ export class MultiplePlots {
             }
           }
         })
-        if (subplot instanceof PlotScatter && subplot.type_ !== "graph2d") {
+
+        if (subplot instanceof ParallelPlot) {
+          subplot.rubber_bands.forEach((rubberBand) => {
+            var actualRubberIndex = rubberBandNames.indexOf(rubberBand.attributeName) 
+            if (actualRubberIndex != -1) {
+              rubberBand.axisMin = rubberBands[actualRubberIndex].axisMin;
```

Comment:
**actual**RubberIndex seems weird. Do you mean "current" ?

Reply:
True